### PR TITLE
perf: Improve homepage performance

### PR DESCRIPTION
**Pull request by soniaklimas**
I want to merge this change because it replaces `background-image` prop with next.js `Image` element.

# Pull Request Checklist

- [ ] Test the changes locally to ensure they work as expected.
- [ ] Document the testing process and results in the pull request description. (Screen recording, screenshot etc)
- [ ] Include new tests for any new functionality or significant changes.
- [ ] Ensure that tests cover edge cases and potential failure points.
- [ ] Document the impact of the changes on the system, including potential risks and benefits.
- [ ] Provide a rollback plan in case the changes introduce critical issues.
- [ ] Update documentation to reflect any changes in functionality.

### CODE REVIEW

File: apps/storefront/messages/en-GB.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "home": {
     "title": "Homepage",
     "description": "Nimara's headless architecture, built on Saleor, provides a flexible, cost-effective solution for online businesses. It’s ready to deploy with minimal setup, supports seamless integration of new providers, and features an intuitive UI for easy storefront customization and management.",
-    "homepage-preview": "Homepage preview"
+    "homepage-preview": "Homepage preview",
+    "hero-banner": "Nimara's hero banner"
```
Comment (GrzegorzDerdak) [inline apps/storefront/messages/en-GB.json:6]:
```suggestion
    "hero-banner-alt": "Nimara's hero banner"
```

**Review by GrzegorzDerdak — APPROVED** [review]
